common/client.ts

```diff
@@ -0,0 +1,33 @@
+import { Client } from "pg";
+
+export default async function createDBClient(dbURI: string) {
+  const client = new Client({
+    connectionString: dbURI,
+  });
+  try {
+    await client.connect();
+    console.log("Successfully connected to database");
+    return client;
+  } catch (err) {
+    console.error("Connection error:", err);
+  }
```

Comment:
throwing away error, caller is going to get undefined. why not have the error be thrown?